pkgs/applications/misc/osmctools/default.nix

```diff
@@ -37,10 +37,10 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Command line tools for transforming Open Street Map files";
-    homepage = ''
-      http://wiki.openstreetmap.org/wiki/Osmconvert
+    homepage = [
+      https://wiki.openstreetmap.org/wiki/Osmconvert
       https://wiki.openstreetmap.org/wiki/Osmfilter
-    '';
+    ];
```

Comment:
Yes, is used when a package has multiple homepages e.g [bridge-utils](https://github.com/NixOS/nixpkgs/blob/3ae7f249dad73ed6debec5733dabb8937debb489/pkgs/os-specific/linux/bridge-utils/default.nix#L25). See also: [avrgcclibc](https://github.com/NixOS/nixpkgs/blob/3ae7f249dad73ed6debec5733dabb8937debb489/pkgs/development/misc/avr-gcc-with-avr-libc/default.nix#L74) [font-droid](https://github.com/NixOS/nixpkgs/blob/3ae7f249dad73ed6debec5733dabb8937debb489/pkgs/data/fonts/droid/default.nix#L55)

Of all the packages in nixpkgs only `osmctools` uses multi-line string